README.md

```diff
@@ -148,6 +150,16 @@ sudo jupyterhub -f ./path/to/jupyterhub_config.py
 
 * [Source Code](oauthenticator/azuread.py)
 
+#### For Azure AD B2C
+
+> See `run.sh` for an [example](./examples/azureadb2c/)
+
+* [Source Code](oauthenticator/azureadb2c.py)
+
+
+
+## 
+
```

Comment:
Just some cleanup here and I think we are good!

Reply:
thanks for the review. Now i have updated the readme as well as the sample config file for better documentation. If everything looks good, can you (or someone else) merge the PR, so we can use the updated package in our project? thanks!

Reply:
Also the new PR included the latest update from master branch, just fyi.